android/app/src/main/java/org/fossasia/openevent/viewmodels/FeedFragmentViewModel.java

```diff
@@ -0,0 +1,51 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+import android.content.Context;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.api.APIClient;
+import org.fossasia.openevent.data.facebook.FeedItem;
+import org.fossasia.openevent.fragments.FeedFragment;
+import org.fossasia.openevent.utils.ConstantStrings;
+import org.fossasia.openevent.utils.SharedPreferencesUtil;
+
+import java.util.List;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+
+/**
+ * Created by mayank on 25/12/17.
+ */
+
+public class FeedFragmentViewModel extends ViewModel {
+
+    private MutableLiveData<List<FeedItem>> feedLive;
+
+    public LiveData<List<FeedItem>> getFeed(Context context, FeedFragment feedFragment) {
```

Comment:
You cannot hold an instance of any ```View```, ```Fragment``` or ```Context``` inside the ```ViewModel``` since it would cause memory leaks.